### DISCUSSION TITLE
Problem installing opendrift

### DISCUSSION THREAD

**Discussion by fran954** [OpenDrift/opendrift#1001] [Q&A]
Hi, I'm new to opendrift and I can't install it. Please, does anyone know how to solve this problem?
Thanks for your time

(base) C:\Users\w7\opendrift> conda config --add channels conda-forge
Warning: 'conda-forge' already in 'channels' list, moving to the top

(base) C:\Users\w7\opendrift>conda env create -f environment.yml
Collecting package metadata (repodata.json): failed

UnavailableInvalidChannel: The channel is not accessible or is invalid.
  channel name: conda-forgeh
  channel url: https://conda.anaconda.org/conda-forgeh
  error code: 404

You will need to adjust your conda configuration to proceed.
Use `conda config --show channels` to view your configuration's current state,
and use `conda config --show-sources` to view config file locations.

(base) C:\Users\w7\opendrift>conda activate opendrift
Could not find conda environment: opendrift
You can list all discoverable environments with `conda info --envs`.

**Comment by knutfrode** [comment]
Hi,

It says the channel "conda-forgeh" is not found, but the "h" at the end should not be there.
Did you perhaps accidentally modify the environment.yml file?
It should look like this:
https://github.com/OpenDrift/opendrift/blob/master/environment.yml

You can check with `git status` if you have modified any files in the repository.

**Comment by fran954** [comment]
Hi!
Yes ! check my files and delete a previous version of conda... and the problem is solved. Thank you very much! :)
